### PR TITLE
Temporarily store requests in Redis.

### PR DESCRIPTION
**Pull request by aliou**
This will allow us to have a better visibility on what's going on, and to re-run the requests locally.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,6 +18,11 @@ router.use((req, res, next) => {
   next();
 });
 
+router.use((req, res, next) => {
+  client.set(Date.now(), JSON.stringify(req.body));
```
Comment (BachirC) [inline index.js:22]: You can't add this line in the `router.use` block above ?

Comment (aliou) [inline index.js:22]: I like having middleware separated. I can merge them if you want.

Comment (BachirC) [inline index.js:22]: Alright ! Keep it this way.

**Comment by BachirC** [comment]
👍

**Pull request merged**